src/sfctl/tests/help_text_test.py

```diff
@@ -0,0 +1,364 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Tests to ensure that -h does not return error when run. Also ensures that help
```

Comment:
Nit, don't use "ensure". It's easier to say "Test -h does not return error". Just a readability improvement if we keep the same voice throughout all the comments.